<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C6C1BD</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainChatWidget">
     <widget class="QTextEdit" name="chatDisplay">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>53</y>
        <width>355</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#F2F2F7;
color: #26313E; 
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>580</y>
        <width>70</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #6C5952</string>
      </property>
      <property name="text">
       <string>Ï†ÑÏÜ°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="groupChatButton">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>14</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #9C857B</string>
      </property>
      <property name="text">
       <string>Í∑∏Î£πÏ±ÑÌåÖ</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="messageInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>280</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#F2F2F7;
color: #26313E; 
</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentRoomLabel_2">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>12</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ìú¥Î®ºÏòõÏ≤¥</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LANssenger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="userListButton">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>14</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #9C857B</string>
      </property>
      <property name="text">
       <string>Ïú†Ï†Ä Î™©Î°ù</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="groupChatWidget">
     <widget class="QPushButton" name="userListButton2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>14</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #9C857B</string>
      </property>
      <property name="text">
       <string>Ïú†Ï†Ä Î™©Î°ù</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentRoomLabel">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>12</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ìú¥Î®ºÏòõÏ≤¥</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LANssenger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainChatButton">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>14</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #9C857B</string>
      </property>
      <property name="text">
       <string>Ï†ÑÏ≤¥Ï±ÑÌåÖ</string>
      </property>
     </widget>
     <widget class="QLabel" name="publicRoomLabel">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>55</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ìú¥Î®ºÏòõÏ≤¥</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>üì£Public Chat</string>
      </property>
     </widget>
     <widget class="QListWidget" name="roomListWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>81</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#F2F2F7;
color: #26313E; 
</string>
      </property>
     </widget>
     <widget class="QLabel" name="privateRoomLabel">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>312</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ìú¥Î®ºÏòõÏ≤¥</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>üîêPrivate Chat</string>
      </property>
     </widget>
     <widget class="QListWidget" name="privateRoomListWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>340</y>
        <width>361</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#F2F2F7;
color: #26313E; 
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createRoomButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>361</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans KR</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #6C5952</string>
      </property>
      <property name="text">
       <string>Create Chat</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ïó∞Í≤∞Ï§ë..</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
